testing/trino-tests/src/test/java/io/trino/tests/TestPartialFlushingOrderByQueries.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests;
+
+import io.trino.Session;
+import io.trino.SystemSessionProperties;
+import io.trino.plugin.tpch.TpchPlugin;
+import io.trino.testing.AbstractTestOrderByQueries;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.QueryRunner;
+
+import static io.trino.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static io.trino.testing.TestingSession.testSessionBuilder;
+
+public class TestPartialFlushingOrderByQueries
```

Comment:
most of the queries in `AbstractTestOrderByQueries` doesn't have limit.

It would be better to write few explicit test cases here.

`TestPartialFlushingOrderByQueries` -> `TestPartialFlushingOrderByWithLimitQueries`

Reply:
Done